### DISCUSSION TITLE
Which 🇮🇹 island is famous for its lemon groves and Limoncello liqueur?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#678] [Galaxy Trivia] [answered]
**Bonus:** List the name and main ingredients of the salad which originated here.

**Comment by DevElkami** [comment]
Island of Capri.

Ingredients are : olive, tomatoes, mozzarella and fresh basil

**Comment by SANKARAMDAS** [comment]
Island of Capri
Limoncello is the Italian word referring to an intensely lemon-flavored liqueur most famously associated with and produced in Sorrento, the Amalfi coast, and the island of Capri, but also very popular throughout all of Italy.

Ingredients: tomatoes, mozzarella, olive, and fresh basil

**Comment by peterbaccaro** [comment, marked as answer]
The traditional Limoncello is made from the fragrant lemons of the Amalfi Coast, the Sorrento Peninsula, the island of **Capri** and Cilento, according to precise rules that define the origin and production methods.

**Caprese Salad** or **Insalata Caprese** made with **tomato**, **mozzarella** and **basil** is the salad that originates from Capri.
